test/Psr7ServerRequestTest.php

```diff
@@ -296,12 +314,25 @@ public function testFromZend($uri, $method, $headers, $body, $query, $post, $fil
         // post
         $this->assertEquals($post, $psr7Request->getParsedBody());
         // files
-        foreach ($psr7Request->getUploadedFiles() as $name => $upload) {
```

Comment:
I had to change this because this was incorrect and buggy also I added test for uploading one file so this had to be changed.